### ISSUE TITLE
Reference Player's Manual in the Preferences Panel

### ISSUE THREAD

**Issue by hmglasgow** [endless-sky/endless-sky#8057]
### Location

In game

### Describe the issue

I recently discovered that I can stop my ship with the SHIFT+DOWN keys. This was in the Player's Manual but not in the Preferences Panel. I get that it's not practical to list every control in the panel, but I feel it would be useful to point new players to this reference. There's a wealth of information in the manual that's not immediately obvious when playing Endless Sky.

### Expected content

![Controls](https://user-images.githubusercontent.com/102213051/210685990-c85d01a4-c069-4037-b2c2-6b00fcfb5097.png)

**Comment by Zitchas** [comment]
That's probably a good idea. Alternatively, having a "help" or "info" button somewhere on here that opens a panel with the manual in it. We could probably copy in the manual document, which would be better for offline players.

**Comment by mOctave** [comment]
Along a similar line, I'm working on adding loading hints, which would provide some information: https://github.com/endless-sky/endless-sky/pull/7958

**Comment by quyykk** [comment]
I don't think we need to million places where we link to the manual. For now having it in credits is probably enough

**Comment by wolfkin** [comment]
seems similar to #7938

I don't think we need to have a bunch of links to the manual but credits isn't exactly where one first goes to look for that type of help. There's room for another tab in preferences and since the game doesn't have a lot of areas you can look for information. Preferences is basically where a new player would look anyway. Heck I only found the manual by googling.

**Comment by lumbar527** [comment]
What if you mention the manual in the `basic 1` help message?
